### PR TITLE
Feature/database

### PR DESCRIPTION
**Pull request by lauriealiu**
Basic UI

### CODE REVIEW

**Review by lauraehiller — CHANGES_REQUESTED** [review]
Hey Laurie, everything looks good!  I'll approve once the merge conflicts are resolved!

**Comment by lauriealiu** [comment]
I think I've resolved the conflict!

**Review by lauraehiller — APPROVED** [review]
LGTM!

**Pull request merged**
